release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -118,3 +175,15 @@ To view bug fixes, see the GitHub milestone for Swan Lake <VERSION> of the repos
 #### Ballerina Packages Updates
 
 ### Breaking Changes
+
+- Unused variables declared with `var` where the inferred type includes error result in a compilation error.
```

Comment:
```suggestion
- Unused variables declared with `var` in which the inferred type includes an error result in a compilation error.
```

Reply:
"in which" isn't quite correct here. The variable is is what we are talking about. Can we say "of which"?

Re: "an error", we are talking about the type here. Will emphasize error maybe.

Reply:
+1

Reply:
Updated in 37da79a.